src/batch-client/index.ts

```diff
@@ -1,4 +1,5 @@
 import DataLoader from 'dataloader';
+import { Agent } from 'https';
```

Comment:
this is nodejs specific package, not sure we should use this in web code

Reply:
Removed